### PR TITLE
fix: include Skia project condiction

### LINKED ISSUE #5695: Cross-runtime dotnet template is faulty
## Current behavior

The Skia project isn't included in the solution when creating a cross-runtime library with `dotnet new unolib-crossruntime`.

In the code, it looks like the WASM project is also using the wrong condition ('Android' instead of 'WebAssembly'):
https://github.com/unoplatform/uno/blob/fd70d8bd6a98d958db27023748f06423f5c53501/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/template.json#L41-L48

## Expected behavior

Skia project is included, correct conditions used

## Workaround

Manually include the Skia project in the solution after it's been created.

## Environment

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by Jadhielv**
GitHub Issue (If applicable): closes #5695

## PR Type

What kind of change does this PR introduce?

- Bugfix

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/template.json [hunk 1]
```diff
@@ -43,9 +43,13 @@
       "path": "UnoCrossRuntimeLib\\UnoCrossRuntimeLib.csproj"
     },
     {
-      "condition": "Android",
+      "condition": "WebAssembly",
```
Comment (Youssef1313) [inline src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/template.json:46]: @jeromelaban Do we use "Wasm" or "WebAssembly" here?

Comment (jeromelaban) [reply src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/template.json:46]: `WebAssembly` is fine. The shortname may be wasm:

https://github.com/unoplatform/uno/blob/540399456be9e6caf7d7616fdc438ee7ea81cb89/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-net6/.template.config/dotnetcli.host.json#L4-L7

It's fine like this.

**Review by jeromelaban — APPROVED** [review]
Thanks for your contribution!

**Comment by Jadhielv** [comment]
You're welcome @jeromelaban

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**